new/LIT7222QeneAgbertihu.xml

```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7222QeneAgbertihu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አመ፡ አግብርቲሁ፡ መሀከ፡ እግዚአብሔር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAma ʾagbǝrtihu mahaka ʾƎgziʾabǝḥer (Qǝne possibly of wāzemā-type)</title>
+                <title xml:lang="en" corresp="#t1">When God took care of his servants…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne possibly of the wāzema-type according to the classification in 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>.
```

Comment:
The same as 7223

Reply:
Here there are more potential qene types that could come in question. One may consider a line break after እምሐቅል፡ and እምሐቅል፡. One can also guess if the rhyme upon -**al** can be considered (but I am not sure if the vowel was also used for creating rhyme, I saw rather that it was only the consonant).